stats/lib/stats-fetchers.js

```diff
@@ -472,12 +472,12 @@ export const fetchAllocatorsRSRSummary = async (pgPools, filter) => {
 
 /**
  * Fetches the retrieval stats summary for a single allocator for given date range.
- * @param {PgPools} pgPools
+ * @param {FastifyPg} pg - Fastify pg object with database connections
```

Comment:
Can we keep the name `pgPools` to keep the diff smaller?

```suggestion
 * @param {FastifyPg} pgPools - Fastify pg object with database connections
```

Same comment applies to all other similar places too.

Reply:
Alright, noted